src/libsystemd/sd-device/device-enumerator.c

```diff
@@ -662,10 +662,8 @@ static int enumerator_add_parent_devices(
                         continue;
 
                 r = device_enumerator_add_device(enumerator, device);
-                if (r < 0)
```

Comment:
I kept skipping this one when running Coccinelle since I wasn't completely sure how to _elegantly_ reformat it, thanks for taking care of this!